app/Services/RatingsService.php

```diff
@@ -4,13 +4,12 @@
 
 class RatingsService {
 
-  public function updateRatings($request)
+  public function updateRatings($user, $request)
```

Comment:
Reminder to add a test for this class. One thing to consider is whether we want to pass in request here. Any pros and cons you can think of?

Reply:
The first con I can think of passing the entire requests is that it makes it harder to test, since you may have to know what the properties the request has.
It may be simpler to pass the exact data (rating).

Thanks for the reminder, will change this.